packages/golden-layout/src/LayoutManager.ts

```diff
@@ -890,6 +931,7 @@ export class LayoutManager extends EventEmitter {
     }
 
     if ($.isPlainObject(contentItemOrConfig) && contentItemOrConfig.type) {
+      assertNotNull(parent);
```

Comment:
All of the content item constructors called in `createContentItem` require `parent` based on their types, so seemed like a good option to assert this here to allow making the type required in `createContentItem` signature.

Reply:
Well looks like this actually fails e2e tests. Going to put back in draft